platform-configs/include/pcengines.robot

```diff
@@ -42,6 +42,7 @@ ${DMIDECODE_TYPE}=                          Desktop
 # Supported test environments
 ${TESTS_IN_FIRMWARE_SUPPORT}=               ${TRUE}
 ${TESTS_IN_UBUNTU_SUPPORT}=                 ${TRUE}
+${TESTS_IN_METATB_SUPPORT}=                 ${TRUE}
```

Comment:
Should probably add this and `TRENCHBOOT_SUPPORT` to default.robot and set to FALSE otherwise those tests would fail with undefined variable exception.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/compare/5163e1a22995279403dcc7f277ea309cd4ac79b3..f3c14936b5b14706e5efa3208f4544fe2ee65cd3#diff-07c15f12435114d3af82c6b0f99da527081f7f6bb6455aee02cbdf868630f838